### PR TITLE
feat(docs): add ecosystem table - I80

### PR DESCRIPTION
**Pull request by daniloff200**
# Issue #80 
<OVERALL SUMMARY OF PULL REQUEST>

### Changes
- <INSERT DESCRIPTION OF CHANGES>
  - <SUB NOTES OF DESCRIPTIONS>
- <SUMMARIZE ALL COMMITS IN PULL REQUEST>

### Flags
- <DESCRIBE ISSUES OR HOLDS FOR REVIEWERS>

### Related Issues
- Issue #<NUMBER HERE>
- Pull Request #<NUMBER HERE>

### CODE REVIEW

**Comment by daniloff200** [comment]
I see u, @irmerk  :) 
Selection_022

Okay,  there's some draft version of table. Please, don't look at package version, that's just an example, also, hyperlinks in project name will be added.

I'm open to any suggestions / improvements here :) 
Just want to create one final and approved version, and then, add it to other repos

**Comment by irmerk** [comment]
This is great looking so far @daniloff200! 

Some initial thoughts: 
- `slate-markdown-transform` has essentially been moved into `markdown-transform`, correct @dselman?
  - If that's the case, it can be removed from this table.
- I think `concerto` should be below `ergo` and `cicero`.
- The sections (`Core`, `Cicero`, `Markdown`, etc.) should probably be in the `Projects` column.
  - Or even possibly another column to the left of `Projects`, which would be called `Projects` and `Projects` will be changed to maybe `Libraries`?

Added some others to review for more input.

**Comment by daniloff200** [comment]
And now we're here :) 
Waiting for some feedback from the team !

**Comment by irmerk** [comment]
> And now we're here :)
> Waiting for some feedback from the team !

Go ahead on the changes I mentioned for now while we wait for further feedback.

**Comment by daniloff200** [comment]
@irmerk 
Boom ! :1st_place_medal: 

I have 2 options for  

> The sections (Core, Cicero, Markdown, etc.) should probably be in the Projects column.
Or even possibly another column to the left of Projects, which would be called Projects and Projects will be changed to maybe Libraries?

Selection_025


1) Just copypaste section name on each line
2) Do it only for the first time

**Comment by irmerk** [comment]
I think option 2 looks cleaner. Also, if it's possible to center and bold the projects maybe.

**Comment by jeromesimeon** [comment]
@daniloff200 @irmerk What's the issue number for this?

I like how it looks, but I get confused by a few things in the table (e.g., what is `Project` ?). I'm thinking the following might be a more accurate reflection of the different libraries/packages:

- Core Libraries:
```
| Project | Package Name | Version | Description |
--------------------------------------------------
| Cicero   | cicero-core   | 0.13.5 | Templates Core |
|          |  ....         | ...    | ...  |
| Concerto | concerto-core | 0.81.1 | Concerto core |
|          | concerto-tools | 0.81.1 | Concerto code generation |
|          |  ....         | ...    | ...  |
| Ergo     | ergo-compiler | 0.9.4 | Ergo compiler |
|          | ergo-engine   | 0.9.4 | Ergo engine |
|          | ergo-test     | 0.9.4 | Ergo testing |
|          | ergo-cli      | 0.9.4 | Ergo command line |
| Markdown | markdown-common | 0.4.7 | ... |
|          | markdown-cicero | 0.4.7 | ... |
|          |  ....         | ...    | ...  |
```

- UI Components:
```
| Project | Package Name | Version | Description |
--------------------------------------------------
| Markdown Editor | cicero-ui | ... |
| Cicero UI | cicero-ui | ... |
```

- Template Editors:
```
| Project | Published at | Description |
--------------------------------------------------
| Template Studio v1 | template-studio | ... |
| Template Studio v2 | template-studio-v2 | ... |
| VSCode extension | cicero-vs-code-extension | ... |`
``

- Public templates and models:
```
| Project | Published at | Description |
--------------------------------------------------
| Models | ... |
| Template Library | ... |
```

**Comment by irmerk** [comment]
@4katie and @Michael-Grover any thoughts?

**Comment by Michael-Grover** [comment]
@jeromesimeon I like your suggestion. Where would techdocs go?

**Comment by daniloff200** [comment]
oh well, I also have one question.

For example, in techdocs repo, should I also add techdocs in table, or, we gonna exclude the repo from the table, when this table located is in this repo? :)

Or, ecosystem table will be exactly the same in all repos?

**Comment by irmerk** [comment]
> Or, ecosystem table will be exactly the same in all repos?

I think this should be within the Accord Project section of each README (about half way down in [cicero's](https://github.com/accordproject/cicero) README), so it should be on all the repos the exact same, not omitting the one it is currently in.

**Comment by jeromesimeon** [comment]
> > Or, ecosystem table will be exactly the same in all repos?
> 
> I think this should be within the Accord Project section of each README (about half way down in [cicero's](https://github.com/accordproject/cicero) README), so it should be on all the repos the exact same, not omitting the one it is currently in.

Can automatically generate that list? If not, this is certainly overkill and will likely not be maintained properly. I would prefer one single place for that information if possible.

**Comment by jeromesimeon** [comment]
> @jeromesimeon I like your suggestion. Where would techdocs go?

Good question. I think a link do the _published_ version of the documentation would be the most useful. We should have that link if we don't have it already.

**Comment by jeromesimeon** [comment]
> @jeromesimeon it's issue #80

Thanks @irmerk @daniloff200  I update the issue description. A lot of this discussion might have been best in the issue itself, rather than in the PR.

**Comment by daniloff200** [comment]
@jeromesimeon  @irmerk 
Selection_029
Selection_030

I'm not completely sure, that I understand all the ideas from @jeromesimeon, but, here's how it looks now, and also, about Published date - where I can found that?

Also, gonna fix all the badges with versions, and all hyperlinks to all Projects and Packages.

**Comment by jeromesimeon** [comment]
> I'm not completely sure, that I understand all the ideas from @jeromesimeon, but, here's how it looks now, and also, about Published date - where I can found that?
> Maybe I missed some items in the table, please, correct me, if so
> Also, gonna fix all the badges with versions, and all hyperlinks to all Projects and Packages.

hi @daniloff200 That looks exactly like what I had in mind! Some additional feedback on your snapshots:

66685408-1e059a80-ec85-11e9-8af2-a88c5c19e442

66685403-1b0aaa00-ec85-11e9-9342-b911df3f4259

**Comment by daniloff200** [comment]
Hey, @jeromesimeon , looks like I've done with that
check this out: https://github.com/daniloff200/techdocs/tree/ecosystem-table-and-badges

One question, that left unclear for me, about the actual link to documentation repo. Where I should put it, and, it's better to use github repo, or, to the https://docs.accordproject.org/ ?

File: README.md [hunk 1]
```diff
@@ -37,6 +37,204 @@ Accord Project is an open source, non-profit, initiative working to transform co
...
+    <td headers="co3 c1"> <b>0.13.4</b></td>
+    <td headers="co5 p2">Web UI for creating, editing and testing Accord Project templates</td>
+  </tr>
+  <tr>
+    <td headers="co1 c1"><a href="https://github.com/accordproject/template-studio-v2">Template Studio v2</a></td>
+    <td headers="co3 c1"> <b>0.13.4</b></td>
+    <td headers="co5 p2">Web UI for creating, editing and testing Accord Project templates</td>
+  </tr>
+   <tr>
+    <td headers="co1 c1"><a href="https://github.com/accordproject/cicero-vscode-extension">VSCode extension</a></td>
```
Comment (irmerk) [inline README.md]: Capitalize `Extension`?

**Comment by daniloff200** [comment]
@irmerk  all comments were applied :)
I hope, that it's last bunch of tweaks, and I will be finally able to create PRs with such table to all repos!

**Pull request merged**